app/styles/identity-card.module.css

```diff
@@ -0,0 +1,35 @@
+

```

Comment:
NIT: extra spacing here

Reply:
Done with changes...can you please check latest comment @satyam73